### PR TITLE
added logger

### PR DESCRIPTION
**Pull request by tzufgoogle**
Added a logger file and logs to geo_computation

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 /bazel-*
 **__pycache__**
 .pytype
+Manhattan_data.txt
+Pittsburgh_data.txt
```
Comment (jasonbaldridge) [inline .gitignore:5]: Please remove these from gitignore, and have a working area outside of your repository for working with data like this.

File: geodata.log [hunk 2]
```diff
@@ -0,0 +1,129 @@
+finished Pittsburgh
+Process ForkPoolWorker-16:
+Traceback (most recent call last):
```
Comment (jasonbaldridge) [inline geodata.log:3]: This file shouldn't be checked in.

File: cabby/logger.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Function to create logger.'''
+
+import logging
+from logging.handlers import RotatingFileHandler
+
+
+def create_logger(save_path,loggername):
```
Comment (jasonbaldridge) [inline cabby/logger.py:21]: need space after comma, e.g. (save_path, loggername)

Also, a brief docstring would be good.

File: .gitignore (hunk 1 above)
Comment (tzufgoogle) [reply .gitignore]: done

File: cabby/logger.py (hunk 3 above)
Comment (tzufgoogle) [reply cabby/logger.py:21]: done

File: geodata.log (hunk 2 above)
Comment (tzufgoogle) [reply geodata.log:3]: done

**Comment by tzufgoogle** [comment]
Made all the requested changes.

**Review by tzufgoogle — COMMENTED** [review]
made all the requested changes

**Pull request merged**